boards/nxp/imx95_evk/imx95_evk_mimx9596_m7_defconfig

```diff
@@ -13,3 +13,4 @@ CONFIG_XIP=y
 CONFIG_MBOX=y
 CONFIG_MBOX_INIT_PRIORITY=0
 CONFIG_ARM_SCMI=y
+CONFIG_SPI=y
```

Comment:
You probably don't want this is in the `defconfig`?

Reply:
May I have a query here? What the recommended configuration for defconfig?
Should we keep the minimum options enabled in defconfig and let user enable what they want to use, or should we enable what the board supports in default?

Thanks a lot.

Reply:
`defconfig` will be include the features in every application that gets built. You probably don't want to enable `SPI` as some of the applications might not be interested in this feature.

Reply:
Thank you so much. Understand now.
I will remove it.